TestKit/Sources/TestKit/DateFormatter+Helpers.swift

```diff
@@ -0,0 +1,23 @@
+import Foundation
+
+/// DateFormatter Extensions
+///
+extension DateFormatter {
+
+    /// Date And Time Formatter. Converts String to Date type
+    ///
+    static public func dateFromString(with dateString: String) -> Date {
+
+        let formatter = DateFormatter()
+
+        formatter.locale = Locale(identifier: "en_US_POSIX")
+        formatter.timeZone = TimeZone(identifier: "GMT")
+        formatter.dateFormat = "yyyy'-'MM'-'dd'T'HH:mm:ss"
```

Comment:
Suggestion: I'm thinking maybe it's more flexible to inject time zone and date format in the parameters of this function, in case we need to test with some other timezone or input a different format. To make it convenient to use, they can go with default values so that it's optional to define them when calling the method:

```Swift
static public func dateFromString(with dateString: String, timeZone: TimeZone? = .init(identifier: "GMT"), dateFormat: String = "yyyy'-'MM'-'dd'T'HH:mm:ss") -> Date {
    // implementation using the parameters
}
```

Reply:
> I'm thinking maybe it's more flexible to inject time zone and date format in the parameters of this function, in case we need to test with some other timezone or input a different format.

Thanks for the suggestion! That sounds great.